.github/workflows/sql-odbc-main.yml

```diff
@@ -13,14 +13,15 @@ env:
   CI_OUTPUT_PATH: "sql-odbc/ci-output"
   ODBC_LIB_PATH: "./build/odbc/lib"
   ODBC_BIN_PATH: "./build/odbc/bin"
-  ODBC_BUILD_PATH: "./build/odbc/build"
-  AWS_SDK_INSTALL_PATH: "./build/aws-sdk/install"
+  ODBC_BUILD_PATH: "./build/odbc/cmake"
+  VCPKG_X64_INSTALL_PATH: ".\src\vcpkg_installed\x64-windows"
+  VCPKG_X86_INSTALL_PATH: ".\src\vcpkg_installed\x86-windows"
```

Comment:
No need to double backslash `\\` or forward slash `/` here?

Reply:
Some magic? Strange I would think it wouldn't work either now that I see it, but the installer failed before I made this change. I updated to use double backslashes but I'm not sure why they aren't required.